README.md

```diff
@@ -92,6 +96,10 @@ You'll need to create a `.env` file in the root of the repo, with the following
 
 `AIRTABLE_API_KEY` - api key for accessing the airtable with youtube video references
 
+`CRM_IMPOSTER_EMAIL` - when using SKIP_AUTH, this is the e-mail used to generate an access cookie
```

Comment:
Can we add `SKIP_AUTH` here, and mention that if `true` the backend will authenticate automatically (user no longer needs to manually visit `localhost:4200?login=accessToken`), and the token will be printed in the console, which can be pasted it into headers of requests, etc.

We should also open an issue in `labs-zap-search` to remove the `localhost:4200?accessToken=test` instruction from the README.md there.

Reply:
Good catch! Will do. Thank you.